docs/src/index.md

```diff
@@ -10,60 +10,33 @@ contact: support@civitas.ca
+MyCivitas is a user-friendly, cost-effective asset management platform designed specifically for smaller, budget-conscious municipalities.  Leveraging GIS technology, it seamlessly caters to both the strategic planning and day-to-day operational requirements of asset management.
 
-Please note that support is not included in the hosting fee. A popular option for small communities is to sign an annual support contract. Please inquire (support@civitas.ca) and we will connect you with a team member near you
+Users can view, locate and query their asset inventory data. MyCivitas calculates and reports Asset Management related information including the condition, replacement cost, life expectancy, probability of failure, consequence of failure and associated risk related to assets.  MyCivitas also includes a Computerized Maintenance Management Software (CMMS) module to create and manage work orders.
 
-**Enjoy Asset Management:** Practice effective and sustainable asset management on our affordable platform to benefit your community.
-
-### Disclaimer
-
-<div class="admonition warning">
-The software provided by this project is provided 'as is'. All information provided
-within the platform should be independently verified before using as the basis for
-action. The contributors and developers of this platform take no responsibility
-for any loss of revenue, life, physical harm or any other adverse outcome that may
-occur as a result of the use of this platform.
-</div>
+MyCivitas uses QGIS, a free open-source desktop GIS and is integrated with Mergin Maps for recording information on site using a mobile app.
 
 <!-- ### Purpose -->
 
-### Scope of project
+## Where to start
 
-Civitas was created as a way to present the captured data on a user-friendly platform, allow staff to query/analyze/understand said data, and perform various asset management-related calculations to assist in capital planning.
+We recommend the following three-step approach: 
 
 ![Scope](./img/scope-1.png)
 
 LandInfo developed a three-step methodology to support smaller communities in the development of their asset management plans:
 
-**Step 1: Asset Register:** The first step is creating an asset register. Assets are divided into classes including water supply, wastewater collection, stormwater collection, transportation, facilities, fleet and natural assets.
+## Step 1: Asset Register
 
-Assets are recorded from existing electronic data, hard-copy maps and reports. The remaining unknown assets are recorded through field surveys using Mergin Maps, an open-source mobile application. Civitas Asset Management, an open-source GIS-based application is used to capture the data in a predefined data model.
+Assets are captured in MyCivitas’s predefined data structure using QGIS.  Typically, data is obtained from existing electronic data, hard-copy maps and reports. The remaining unknown assets are recorded through field surveys using Mergin Maps, an open-source mobile application.
```

Comment:
@ketankartoza, when you make another PR this will need to be addressed. This PR was merged before I had a chance to review the language/spelling.
```suggestion
Assets are captured in MyCivitas’ predefined data structure using QGIS.  Typically, data is obtained from existing electronic data, hard-copy maps and reports. The remaining unknown assets are recorded through field surveys using Mergin Maps, an open-source mobile application.
```

There are also instances of double spaces that need to be addressed.